client/src/components/QandA/QuestionsList/QuestionCard.jsx

```diff
@@ -64,46 +66,61 @@ class QuestionCard extends React.Component {
     }
   }
 
+  onAccordionClick() {
+    this.setState({
+      accordionDisplay: !this.state.accordionDisplay
+    });
+  }
+
   render() {
     return (
       <div id='QA-question-card'>
-        <QuestionDetails
-          asker={this.props.asker}
-          date={this.props.date}
-          helpful={this.props.helpful}
-          reported={this.props.reported}
-          question_id={this.props.question_id}
-          product_name={this.props.product_name}
-        />
-        <div className='QA-symbol QA-question-symbol'>Q:&nbsp;&nbsp;</div>
-        <QuestionBody body={this.props.body}/>
-        {Object.keys(this.props.answers).length > 0 ?
-          <>
-            <div className='QA-symbol QA-answer-symbol'>A:&nbsp;&nbsp;</div>
-            <Answers
-              answers={this.props.answers}
-              key={this.props.question_id * 3}
-              answerCounter={this.state.answerCounter}
-              showMore={this.state.showMoreBtn}
-              collapse={this.state.showCollapseBtn}
-            />
-            {
-              !this.state.showMoreBtn ? null
-                : !this.state.showCollapseBtn
-                  ?
-                  <>
-                    <button onClick={this.onSeeMoreClick}>See more answers</button>
-                    <br/><br/>
-                  </>
-                  :
-                  <>
-                    <button onClick={this.onCollapseClick}>Collapse answers</button>
-                    <br/><br/>
-                  </>
-            }
-          </>
-          : null
-        }
+        <div id='QA-question-card-question'>
+          <div className='QA-symbol QA-question-symbol'>Q:&nbsp;&nbsp;</div>
```

Comment:
It seems like QA-question-symbol isn't an actual class in your CSS? I think you just need QA-symbol here?